astronomer/providers/snowflake/hooks/snowflake.py

```diff
@@ -81,16 +83,28 @@ def run(  # type: ignore[override]
                 conn.commit()
         return self.query_ids
 
-    def check_query_output(self, query_ids: List[str]) -> None:
+    def check_query_output(
+        self, query_ids: List[str], handler: Callable | None = None, return_last: bool = True
+    ) -> Optional[Union[Any, List[Any]]]:
```

Comment:
Is it possible to specify what `Any` would mean in the Union. Because `List[Any]` would anyway be a subset of` Any`. If it for representing just one record, and not a list can we specify the type of the record so that it gives a better idea of the return type?

Reply:
yeah, `List[Any]` will be cleaner and make sense here

Reply:
It can be list of response or it can be single response or it can be None as well